.travis.yml

```diff
@@ -0,0 +1,20 @@
+os: osx
+osx_image: xcode9.4
+language: node_js
+node_js: "10"
+env:
+  - ELECTRON_CACHE=$HOME/.cache/electron
+  - ELECTRON_BUILDER_CACHE=$HOME/.cache/electron-builder
+
+cache:
+  directories:
+    - node_modules
+    - $HOME/.cache/electron
+    - $HOME/.cache/electron-builder
+
+script:
+  - ./bin/build
+
+branches:
+  except:
+    - "/^v\\d+\\.\\d+\\.\\d+$/"
```

Comment:
Curious: is this for `v[0-9].[0-9].[0-9]` branches? Shouldn't we only run the build script on merges to master?

Reply:
Yes, that is for `v[0-9].[0-9].[0-9]` branches. I suppose it makes sense to only build for merges to master, but it might not hurt to build for pull requests as well as it would work sort of like a "deployment preview". Of course that depends on the build times 😊 

I can change it to master-merges only if you wish

Reply:
👍, let's try it out and see how much travis likes building it each time 😄.

I guess we would look at the travis logs to find the `transfer.sh` url to download the preview? I've enabled this repo on travis now.

Reply:
Yes, unfortunately it requires looking at the logs.

Looking at the logs now it looks like we need to set `GH_TOKEN` for it to work - this might also be a way to make a simple request to GitHubs API to include the release links in a status check or similar (like how Netlify/Heroku does it for deploy previews).